### ISSUE TITLE
Implementing Action Recognition on the badminton dataset

### ISSUE THREAD

**Issue by ashwinvaswani** [SforAiDl/Playground#89]
We can make use of existing implementations of I3D/CSNs from https://github.com/open-mmlab/mmaction

**Comment by ashwinvaswani** [comment]
Yes, I'm currently working on it.  You can join in if you want, I'll ping you the details on slack

**Comment by veds12** [comment]
I am not on the channel. Can you put the link here?

**Comment by ashwinvaswani** [comment]
Yeah I've added the channel link in the base readme
